src/App.css

```diff
@@ -2,4 +2,20 @@
 
 * {
     font-family: 'Montserrat', sans-serif;
+}
+
+.logo {
+  width: 150px;
+  height: 36px;
+  align-self: center;
```

Comment:
You can add a `margin-top: -2px` to position the logo pixel-perfect. Logo and links were still a bit off
![image](https://user-images.githubusercontent.com/9722792/77829540-f6b36480-7122-11ea-97ed-9a22dcdf0f29.png)